.github/dependabot.yml

```diff
@@ -0,0 +1,29 @@
+version: 2
```

Comment:
Just so you know, dependabot will automatically add the [dependencies](https://github.com/python/typeshed/pulls?q=label%3Adependencies)  label with default configs (which I believe is something we'd want, but has only been used for a single PR on typeshed before)
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#labels